keps/sig-multicluster/1645-multi-cluster-services-api/README.md

```diff
@@ -1013,6 +1029,13 @@ Session affinity affects a service as a whole for a given consumer. The derived
 service's session affinity will be decided according to the conflict resolution
 policy.
 
+#### Labels and Annotations
+
+If supported, exporting labels and annotations would affect a `Service` as a whole
```

Comment:
```suggestion
If supported, exporting labels and annotations would affect a service as a whole
```
For consistency with the above sections which do not explicitly mention the `Service` resource - my reading is that a derived `Service` resource is an optional implementation detail which is one component of the "service as a whole".

Reply:
I see this would revert @sftim's prior suggested change in https://github.com/kubernetes/enhancements/pull/4922#discussion_r1836993024